### DISCUSSION TITLE
URL Shortener config generation

### DISCUSSION THREAD

**Discussion by IceToast** [diced/zipline#263] [Ideas]
I would love to see a config generator for the uploaders (sharex, flameshoot) for the url shortner.
The "upload" config doesn't work due to the fact that url shorting uses the api/shorten endpoint.

**Comment by diced** [comment]
Added in https://github.com/diced/zipline/commit/6d49463dad16ce1369dab2b1366ccfe57379b537

**Comment by ItsNoted** [comment]
Is it possible to use a different domain for the url shortner separate from the files domain?
